test/hotel_controller_test.rb

```diff
@@ -6,26 +6,23 @@
     @date = Date.parse("2020-08-04")
   end
   describe "wave 1" do
-    describe "rooms" do
-      it "returns a list" do
-        rooms = @hotel_controller.rooms
-        expect(rooms).must_be_kind_of Array
-      end
-    end
     describe "reserve_room" do
       it "takes two Date objects and returns a Reservation" do
```

Comment:
This is a good test. Nice work! You also need to test edge cases. What if there is no room available?